es6/report-generator.js

```diff
@@ -28,7 +28,8 @@ export function generateFileReport(file, spellingInfo) {
 
     const lineNumber = String(displayBlock.lineNumber);
     const lineNumberPadding = Array(10 - lineNumber.length).join(' ');
-    const linePrefix = `${lineNumberPadding}${lineNumber} |`;
+    const errorIndex = String(error.index);
```

Comment:
is `error.index` definitely the column and not the file index? maybe it is and I just chose bad naming...